src/plugins/ui_core_views/pivot_ui.ts

```diff
@@ -198,8 +198,8 @@ export class PivotUIPlugin extends UIPlugin {
       return EMPTY_PIVOT_CELL;
     }
     if (functionName === "PIVOT") {
-      const includeTotal = args[2] === false ? false : undefined;
-      const includeColumnHeaders = args[3] === false ? false : undefined;
+      const includeTotal = args[2] === false || args[2] === 0 ? false : undefined;
+      const includeColumnHeaders = args[3] === false || args[3] === 0 ? false : undefined;
```

Comment:
```suggestion
      const includeTotal = toBoolean(includeTotal);
      const includeColumnHeaders = toBoolean(includeColumnHeaders);
```
let's have the same behaviour as the formula :) It also seem to support the strings "True" and "FaLsE"

Reply:
Toboolean(undefined) = false, but in this case, if we have undefined, it should mean true. it works in the formulas because there are default values  in `PIVOT.compute` Altough I agree it could/should be more robust.